src/middleware/README.md

```diff
@@ -3,8 +3,8 @@
 If you only need the middleware with access to the `.sign()`, `.verify()` or the receiver’s `.receive()` method, you can use the webhooks middleware directly
 
 ```js
-const Middleware = require('@octokit/webhooks/middleware')
-const middleware = new Middleware({
+import { createMiddleware } from '@octokit/webhooks/middleware'
```

Comment:
This should be 

```ts
import { createMiddleware } from '@octokit/webhooks'
```

With the migration to typescript / ESM exports, I'd export everything at the root package.

Reply:
Also if at all possible, I'd like to avoid any breaking changes. Either we make the previous `require('@octokit/webhooks/middleware')` work and log a deprecation message, or we make the change in the current version, where we make the new export work and start logging deperecation messages for the current usage

Reply:
> This should be
> 
> ```ts
> import { createMiddleware } from '@octokit/webhooks'
> ```
> 
> With the migration to typescript / ESM exports, I'd export everything at the root package.

Applied :)

Reply:
> Also if at all possible, I'd like to avoid any breaking changes. Either we make the previous `require('@octokit/webhooks/middleware')` work and log a deprecation message, or we make the change in the current version, where we make the new export work and start logging deperecation messages for the current usage

I can open a new PR from `master` adding this first step of logging and changing entrey points and I rebase this branch with master once the changes are landed. Do you like the idea?